src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -149,10 +155,39 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseSer
         return ResponseEntity.ok(responseComplaints);
     }
 
+    /**
+     * Get /more-feedback/for-tutor-dashboard/:exerciseId
+     * <p>
+     * Get all the more feedback requests associated to an exercise, that are about the tutor who is doing the request.
+     * @param exerciseId the id of the exercise we are interested in
+     * @return the ResponseEntity with status 200 (OK) and a list of more feedback requests. The list can be empty
+     */
+    @GetMapping("/more-feedback/for-tutor-dashboard/{exerciseId}")
```

Comment:
I don't like this URL scheme at all. It doesn't make sense. Please conform to REST standards here.
One possible improvement would be `/exercises/{exercise-id}/more-feedback-for-tutor-dashboard

While doing that, please also fix the occurrence of @GetMapping("/complaints/for-tutor-dashboard/{exerciseId}")

Reply:
Thank you for the suggestion. I changed endpoints accordingly